src/main/java/duke/UI.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * UI class which handles all the interactions with the user.
+ */
+public class UI {
+
+    private static String logo = " ____        _ _\n"
```

Comment:
```suggestion
    private static final String LOGO = " ____        _ _\n"
```
Would it be good for this constant name to be capitalized? This is mentioned in the [Naming](https://se-education.org/guides/conventions/java/intermediate.html#naming) conventions of the module coding standard.